src/commands/workspace/create.ts

```diff
@@ -25,29 +27,50 @@ export default class Create extends Command {
     chenamespace: cheNamespace,
     devfile: string({
       char: 'f',
-      description: 'path or URL to a valid devfile',
+      description: 'Path or URL to a valid devfile',
       env: 'DEVFILE_PATH',
       required: false,
     }),
     name: string({
-      description: 'workspace name: overrides the workspace name to use instead of the one defined in the devfile.',
+      description: 'Workspace name: overrides the workspace name to use instead of the one defined in the devfile.',
       required: false,
     }),
     start: boolean({
       char: 's',
       description: 'Starts the workspace after creation',
       default: false
     }),
+    debug: boolean({
+      char: 'd',
+      description: 'Debug workspace start. It is useful when workspace start fails and it is needed to print more logs on startup. This flag is used in conjunction with --start flag.',
+      default: false
+    }),
     'access-token': accessToken
   }
 
   async run() {
     const { flags } = this.parse(Create)
+    const ctx: any = {}
+
+    const apiTasks = new ApiTasks()
+    const cheTasks = new CheTasks(flags)
+    const workspaceTasks = new WorkspaceTasks(flags)
+
+    const tasks = new Listr([], { renderer: 'silent' })
+
+    tasks.add(apiTasks.testApiTasks(flags, this))
+    tasks.add(cheTasks.verifyCheNamespaceExistsTask(flags, this))
+    tasks.add(cheTasks.retrieveEclipseCheUrl(flags))
+    tasks.add(cheTasks.checkEclipseCheStatus())
+    tasks.add(workspaceTasks.getWorkspaceCreateTask(flags.devfile, flags.name))
+    if (flags.start) {
+      tasks.add(workspaceTasks.getWorkspaceStartTask(flags.debug))
+    }
+    tasks.add(workspaceTasks.getWorkspaceIdeUrlTask())
 
-    const tasks = this.getWorkspaceCreateTasks(flags)
     try {
-      let ctx = await tasks.run()
-      this.log('\nWorkspace IDE URL:')
+      await tasks.run(ctx)
+      this.log(`Workspace successfully ${flags.start ? 'started' : 'created'}. Workspace IDE URL:`)
```

Comment:
It's not percise to say `started` as we have only sent a request to start the workspace. it might take a minute until workspace state become `running`.

Reply:
What about `Workspace successfully run` ?

Reply:
Please consider: `Workspace has been successfully created. Workspace start request has been sent, workspace will be available shortly`. But up to you.